tools/swift-demangle/swift-demangle.cpp

```diff
@@ -30,12 +30,16 @@
 
 #include <cstdlib>
 #include <string>
-#if !defined(_MSC_VER) && !defined(__MINGW32__)
+#if !defined(_WIN32) && !defined(__MINGW32__)
```

Comment:
Sure - what I'm hoping for is to actually be able to get rid of this whole thing, and just import `<iostream>` for `std::getline`, so this might actually all go!